addons/html_builder/static/src/builder/plugins/clean_for_save_options_plugin.js

```diff
@@ -0,0 +1,26 @@
+import { Plugin } from "@html_editor/plugin";
+
+export class CleanForSaveOptions extends Plugin {
+    static id = "cleanForSaveOptions";
+    resources = {
+        clean_for_save_handlers: this.cleanForSave.bind(this),
+    };
+
+    setup() {
+        this.builderOptions = this.getResource("builder_options");
+    }
+
+    cleanForSave({ root }) {
+        for (const option of this.builderOptions) {
+            const { selector, exclude, clean_for_save_handlers_options } = option;
+            if (!clean_for_save_handlers_options) {
+                continue;
+            }
+            let editingEls = Array.from(root.querySelectorAll(selector));
```

Comment:
Note: you can make an array by doing this too `[...root.querySelectorAll(selector)]` :wink: